### PR TITLE
CAS sign-out

### PR DESCRIPTION
**Pull request by mjweiberg28**
Adding a method (cas_passthrough) to Views to sign the user out of CAS when they click the "start session" button on the first page they view (index.html).

### CODE REVIEW

File: app/templates/error403.html [hunk 1]
```diff
@@ -3,5 +3,5 @@
 {% block body_content %}
 <h1>Permission Denied</h1>
 <h5>Only full-time ITS employees have permission to access this page.</h5>
-<h5>Return to the <a href="{{ url_for('ShiftsView:index') }}">homepage</a>.</h5>
+<h5>Return to the <a href="{{ url_for('ShiftsView:cas_passthrough') }}">homepage</a>.</h5>
```
Comment (c-schwarze) [inline app/templates/error403.html]: is that the route you want to go to? Don't you want to go to the index?

File: app/templates/error500.html [hunk 2]
```diff
@@ -3,5 +3,5 @@
 {% block body_content %}
 <h1>Server Error</h1>
 <h5>Service Desk Sign-In failed.</h5>
-<h6>This page is having trouble loading. Sorry about that! Try returning to the <a href="{{ url_for('ShiftsView:index') }}">homepage</a>.</h6>
+<h6>This page is having trouble loading. Sorry about that! Try returning to the <a href="{{ url_for('ShiftsView:cas_passthrough') }}">homepage</a>.</h6>
```
Comment (c-schwarze) [inline app/templates/error500.html]: is that the route you want to go to? Don't you want to go to the index?

File: app/templates/student_signin.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+{% block body_content %}
+<div class="body-margins-main">
+    <form>
+        <div class="centered">
+            <h3 id="title-bar">Scan ID to Sign In or Out</h3>
+            <h3 class="alert alert-success" id="time-clock-success" style="display: none;">Scan success</h3>
+            <h3 class="alert alert-danger" id="time-clock-no-match" style="display: none;">Scan failed, user not found</h3>
+            <h3 class="alert alert-danger" id="time-clock-fail" style="display: none;">Scan failed, try again</h3>
+            <h3 class="alert alert-danger" id="resource-exhausted" style="display: none;">Error: Server Exhausted. Please wait 1 minute and try again.</h3>
```
Comment (c-schwarze) [inline app/templates/student_signin.html:11]: These alerts should probably be using AJAX. I'm fine if this is a todo for later.

**Review by c-schwarze — CHANGES_REQUESTED** [review]
Micah, want to confirm those routes I commented on? Besides that it looks good.

File: app/templates/error403.html (hunk 1 above)
Comment (mjweiberg28) [reply app/templates/error403.html]: Whoops, yes I want to go to the index. Changing that back.

**Pull request merged**